src/Pos/CompileConfig/Type.hs

```diff
@@ -57,4 +57,5 @@ data CompileConfig = CompileConfig
       -- ^ Block retrieval queue capacity
     , ccPropagationQueueSize          :: !Int
       -- ^ InvMsg propagation queue capacity
+    , ccMaxBlundFileSize              :: !Int
```

Comment:
Why not use memory units here then?
Casual reminder: move `memory-units` into separate project.